jina/peapods/jinad.py

```diff
@@ -0,0 +1,193 @@
+                                uses_files=uses_files,
+                                pymodules_files=pymodules_files,
+                                logger=logger)
+
+    if uses_files:
+        for current_file in uses_files:
+            with open(current_file) as f:
+                result = ruamel.yaml.round_trip_load(f)
+
+            if 'metas' in result and 'py_modules' in result['metas']:
+                _add_file_to_list(_file=result['metas']['py_modules'],
+                                  _file_list=pymodules_files,
+                                  logger=logger)
+
+    return uses_files, pymodules_files
+
+
+class JinadAPI:
+    def __init__(self,
+                 host: str,
+                 port: int,
+                 logger):
+        self.logger = logger
+        self.base_url = f'http://{host}:{port}/v1'
+        self.alive_url = f'{self.base_url}/alive'
+        self.upload_url = f'{self.base_url}/upload'
+        self.pea_url = f'{self.base_url}/pea'
+        self.pod_url = f'{self.base_url}/pod'
+        self.log_url = f'{self.base_url}/log'
+
+        try:
+            import requests
+        except (ImportError, ModuleNotFoundError):
+            self.logger.critical('missing "requests" dependency, please do pip install "jina[http]"'
+                                 'to enable remote Pea/Pod invocation')
+
+    def is_alive(self):
+        import requests
+        try:
+            r = requests.get(url=self.alive_url)
+            return True if r.status_code == requests.codes.ok else False
+        except requests.exceptions.ConnectionError:
+            return False
+
+    def _upload_files(self, uses_files, pymodules_files):
+        import requests
+
+        with ExitStack() as file_stack:
+            files = []
+            if uses_files:
+                files.extend([('uses_files', file_stack.enter_context(open(fname, 'rb')))
+                              for fname in uses_files])
+            if pymodules_files:
+                files.extend([('pymodules_files', file_stack.enter_context(open(fname, 'rb')))
+                              for fname in pymodules_files])
+            if not files:
+                self.logger.info('nothing to upload to remote')
+                return
+
+            headers = {'content-type': 'multipart/form-data'}
+            r = requests.put(url=self.upload_url,
+                             files=files)
+            if r.status_code == requests.codes.ok:
+                self.logger.info(f'Got status {colored(r.json()["status"], "green")} from remote')
+
+    def create(self, kind: str, pea_args: Dict, pod_type: str = 'flow'):
+        import requests
+        try:
+            url = self.pea_url if kind == 'pea' else f'{self.pod_url}/{pod_type}'
+            r = requests.put(url=url,
+                             json=pea_args)
+            return r.json()[f'{kind}_id'] if r.status_code == requests.codes.ok else False
```

Comment:
Keep the return type consistent.
```suggestion
            return r.json()[f'{kind}_id'] if r.status_code == requests.codes.ok else None
```